index.html

```diff
@@ -1,8 +1,100 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang = "en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset = "UTF-8">
+    <meta name = "viewport" content = "width = device-width, initial-scale = 1">
     <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel = "stylesheet" href = "css/style.css">
   </head>
+
+  <body>
+    <h1 id = 'title'>freeCodeCamp Survey Form</h1>
+    <p>Thank you for taking the time to help us improve the platform</p>
+
+    <div>
+      <form id = 'survey-form'>
+        <div>
+          <label for = "name" style = "color: red">Name</label><br>
+          <input Class  =  "large" type = "text" id = "name" placeholder = "enter your name here please">
```

Comment:
- []  The properties of the tags have always to be lowercase letters. I observe that the class properties of the tags in your code starts with an uppercase letter as 'C'. Kindly change all of 'Class' to 'class'.